app/build.gradle

```diff
@@ -18,7 +18,7 @@ android {
 
         // Writing show taps setting fails on higher versions
         //noinspection OldTargetApi
-        targetSdkVersion 22
+        targetSdkVersion 26
```

Comment:
This change unfortunately breaks the Show Taps tile and Google Play no longer allows uploading APKs with `targetSdkVersion` less than 28.